web/src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -356,11 +356,13 @@ export default {
       let extra = Object.keys(meta).filter(
         (x) => !mainFields.includes(x) && x in this.schema.properties,
       );
-      const remove_list = ['citation', 'repository', 'url', 'schemaVersion', 'version', 'id', 'keywords'];
+      const remove_list = ['citation', 'repository', 'url', 'schemaVersion', 'version', 'id', 'keywords', 'schemaKey'];
       extra = extra.filter((n) => !remove_list.includes(n));
       const extra_obj = extra.reduce((obj, key) => ({ ...obj, [key]: meta[key] }), {});
       extra_obj.contributor = _.filter(meta.contributor, (author) => author.schemaKey !== 'Person');
-      // console.log(362, extra_obj);
+      delete extra_obj.assetsSummary.schemaKey;
+      delete extra_obj.assetsSummary.numberOfBytes;
+      delete extra_obj.assetsSummary.numberOfFiles;
```

Comment:
Why remove these values? They are an important part of the asset summary.

Reply:
satra suggested not to display these. should I create a different object with only the key-values to be displayed?

Reply:
> Why remove these values? They are an important part of the asset summary.

satra suggested not to display these. should I create a different object with only the key-values to be displayed?

Reply:
If the intent was not to display them, then this is fine.

@satra, what's the reason to skip showing these values?

Reply:
because they are also in the side panel. we can decide what to use where when we redo the overall design of the landing page.

Reply:
Ah I see. I realized this while discussing design with Jared just now. There is also `numberOfSubjects`; could that similarly go with the number of files/assets? Perhaps with a little mouse icon?